src/modules/pulumi.rs

```diff
@@ -81,24 +81,31 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     }
 }
 
-/// Parse the output of `pulumi version` into just the version string.
+/// Parse and sanitize the output of `pulumi version` into just the version string.
 ///
 /// Normally, this just means returning it. When Pulumi is being developed, it
 /// can return results like `3.12.0-alpha.1630554544+f89e9a29.dirty`, which we
 /// don't want to see. Instead we display that as `3.12.0-alpha`.
 fn parse_version(version: &str) -> &str {
+    let new_version = version.strip_prefix('v').unwrap_or(version);
+
+    let sanitized_version = new_version
+        .strip_suffix("\r\n")
+        .or_else(|| new_version.strip_suffix('\n'))
+        .unwrap_or(new_version);
```

Comment:
This should also work, and would be a bit simpler:
```suggestion
    let sanitized_version = new_version.trim_end();
```

Reply:
Great suggestion! I completely missed fine print of trim_end() including new lines when looking through the docs. I've updated the PR to reflect the suggestion!